### PR TITLE
Added a new level scene :\n

### PR DESCRIPTION
**Pull request by Hhk187**
- Now creating levels is easier

### CODE REVIEW

**Comment by Hhk187** [comment]
Conflicts are dealt with

File: inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		{"default" , new CompressedTexture2D[] {
+			(CompressedTexture2D)GD.Load($"{_PATH}/default/back_ground.png"),
+			(CompressedTexture2D)GD.Load($"{_PATH}/default/middle_ground.png"),
+			(CompressedTexture2D)GD.Load($"{_PATH}/default/for_ground.png")}}
+	};
+
+	/// <summary>
+	/// This function is called whenever the slide down in the editor is changed
+	/// </summary>
+	private void _change_background()
```
Comment (vitormaduro) [inline inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs:39]: Please use `PascalCase` when naming methods (this method should be renamed to `ChangeBackground`)

File: inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+[Tool]
+public partial class parallax_background_blueprint : ParallaxBackground
+{
+
+	/// <summary>
+	/// This export velue adds the ability to modify the enum directly from the editor
+	/// </summary>
+	string bg;
+	[Export(PropertyHint.Enum, "None,default")]
+	string type_of_bg {
```
Comment (vitormaduro) [inline inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs]: Please use `camelCase` when naming variables (this variable should be renamed to `typeOfBg`)

File: inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs (hunk 1 above)
Comment (vitormaduro) [inline inherited_scenes/level_blueprint/level_assets/prallax_background_blueprint/parallax_background_blueprint.cs]: Please use `PascalCase` when naming methods (this method should be renamed to `ChangeBackground`)

**Review by vitormaduro — COMMENTED** [review]
Everything looks good, just fix the names I pointed out and the merge is approved

**Comment by Hhk187** [comment]
vars are re-named

**Review by vitormaduro — APPROVED** [review]

**Pull request merged**